Tests/Integration/DotNet/Azure/Storage/Blobs/AzureBlobStorageTests.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Xunit;
+
+namespace IntegrationTests.Azure.Storage.Blobs
+{
+    [Trait("TestCategory", "Deprecated")]
```

Comment:
I'm not sure about this one. but the "Deprecated" trait ringed some bells.
If there's a good reason to keep this, ignore my comment. Just double checking.

Reply:
We are defining all the classes that are marked as deprecated in the sdk here as well in the tests, so it is easier to identify, follow, filter for v5, etc in the future.